pkg/controller/volume/selinuxwarning/translator/selinux_translator.go

```diff
@@ -61,11 +61,13 @@ func (c *ControllerSELinuxTranslator) SELinuxOptionsToFileLabel(opts *v1.SELinux
 // These labels must be generated by SELinuxOptionsToFileLabel above
 // (the function expects strict nr. of elements in the labels).
 // Since this translator cannot default missing components,
-// the missing components are treated as incomparable and they do not
+// the first three components treated as incomparable when missing and they do not
 // conflict with anything.
```

Comment:
nit: Can you add a comment about what result incomprable leads to? This function returns a boolean value but comparison has three states: conflict, notconflict, incomparable.

Reply:
I tried to rephrase it in a new commit